src/patterns/news/index.njk

```diff
@@ -0,0 +1,114 @@
+---
+title: News
+group: Pages
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/lists/_macro.njk" import onsList %}
+
+How to display, tag, categorise and provide a landing page preview for news articles.
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/landing/index.njk" })
+}}
+
+## When to use this pattern
+
+Use this pattern to upload a news article to a public-facing website.
+
+## How to use this pattern
+
+Each article needs to have a:
+
+- heading
+- summary
+- body text
+- category
+- date
+
+Use the [articles component](/components/articles) to build consistent articles.
+
+The content needs to be clearly written and as short as possible. Use plain language and “frontload” – put the most important word or points first. This ensures that if the user is skimming the article, they read the most important thing and understand it. 
+
+Our house style guide has [advice on writing in plain English](https://style.ons.gov.uk/category/writing-for-the-web/plain-english). You could also ask a content designer to help you rewrite the content so it is clear and helpful for the user.
```

Comment:
Yes, was thinking that this was perhaps too much detail here. It will work with just a mention of the articles component here (as we do on other patterns) and I'll keep all the detail for there